include/anyode/anyode_blasless.hpp

```diff
@@ -1,123 +1,125 @@
 #pragma once
-#include <algorithm>  // std::min
+
 #include <cmath>
 #include <functional>
-#include <anyode/anyode_util.hpp>
 
+#include <anyode/anyode.hpp>
+
+using namespace std;
```

Comment:
we shouldn't have any `using` directives outside namespaces (or we will mess up the whole translation unit of 3rd party code importing this header). Also I personally dislike `using namespace` -- I'm fine with mutiple using statements though (`using namespace` is a bit like `from somelib import *` -- you'd need to know whats in there by heart)